mvpa2/algorithms/hyperalignment.py

```diff
@@ -154,6 +154,10 @@ class Hyperalignment(ClassWithCollections):
                 reference.  If `None`, then the dataset with the maximum
                 number of features is used.""")
 
+    nproc = Parameter(1, constraints=EnsureInt() & EnsureRange(min=1),
+            doc="""Number of processes to use to parallelize the last step of
+                alignment. Requires joblib package.""")
```

Comment:
joblib's n_jobs:
```
n_jobs: int, default: 1
    The maximum number of concurrently running jobs, such as the number
    of Python worker processes when backend="multiprocessing"
    or the size of the thread-pool when backend="threading".
    If -1 all CPUs are used. If 1 is given, no parallel computing code
    is used at all, which is useful for debugging. For n_jobs below -1,
    (n_cpus + 1 + n_jobs) are used. Thus for n_jobs = -2, all
    CPUs but one are used.
```
so -- do not EnsureRange and below check for `nproc != 1`, not `> 1`